<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileSystem</class>
 <widget class="QWidget" name="FileSystem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Filesystem</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Download media locations</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="resource.qrc">:/resource/16x16_ico_png/ico_media_xxx.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Adult</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="edtAdult"/>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="btnAdult">
        <property name="maximumSize">
         <size>
          <width>77</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="resource.qrc">:/resource/16x16_ico_png/ico_media_apps.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Apps</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="edtApps"/>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="btnApps">
        <property name="maximumSize">
         <size>
          <width>77</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="resource.qrc">:/resource/16x16_ico_png/ico_media_console.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Games</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="edtGames"/>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="btnGames">
        <property name="maximumSize">
         <size>
          <width>77</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="resource.qrc">:/resource/16x16_ico_png/ico_media_image.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Images</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="edtImages"/>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="btnImages">
        <property name="maximumSize">
         <size>
          <width>77</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="resource.qrc">:/resource/16x16_ico_png/ico_media_audio.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Music</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="edtMusic"/>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="btnMusic">
        <property name="maximumSize">
         <size>
          <width>77</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="resource.qrc">:/resource/16x16_ico_png/ico_media_movie.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Movies</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLineEdit" name="edtMovies"/>
      </item>
      <item row="5" column="3">
       <widget class="QPushButton" name="btnMovies">
        <property name="maximumSize">
         <size>
          <width>77</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="resource.qrc">:/resource/16x16_ico_png/ico_media_tv.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Television</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLineEdit" name="edtTelevision"/>
      </item>
      <item row="6" column="3">
       <widget class="QPushButton" name="btnTelevision">
        <property name="maximumSize">
         <size>
          <width>77</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="resource.qrc">:/resource/16x16_ico_png/ico_media_other.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Other</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QLineEdit" name="edtOther"/>
      </item>
      <item row="7" column="3">
       <widget class="QPushButton" name="btnOther">
        <property name="maximumSize">
         <size>
          <width>77</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_8">
     <property name="title">
      <string>Download options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QCheckBox" name="chkOverwriteExisting">
        <property name="toolTip">
         <string/>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Overwrite existing files</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="chkDiscardText">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Discard text content</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="chkCheckLowDisk">
        <property name="text">
         <string>Check for low disk before downloading</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Application log files</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="edtLog">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Location for application log files</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnLog">
        <property name="maximumSize">
         <size>
          <width>77</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edtAdult</tabstop>
  <tabstop>btnAdult</tabstop>
  <tabstop>edtApps</tabstop>
  <tabstop>btnApps</tabstop>
  <tabstop>edtGames</tabstop>
  <tabstop>btnGames</tabstop>
  <tabstop>edtImages</tabstop>
  <tabstop>btnImages</tabstop>
  <tabstop>edtMusic</tabstop>
  <tabstop>btnMusic</tabstop>
  <tabstop>edtMovies</tabstop>
  <tabstop>btnMovies</tabstop>
  <tabstop>edtTelevision</tabstop>
  <tabstop>btnTelevision</tabstop>
  <tabstop>edtOther</tabstop>
  <tabstop>btnOther</tabstop>
  <tabstop>chkOverwriteExisting</tabstop>
  <tabstop>chkDiscardText</tabstop>
  <tabstop>chkCheckLowDisk</tabstop>
  <tabstop>edtLog</tabstop>
  <tabstop>btnLog</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
